Assets/Scripts/Runtime/Scriptable Objects/PlaceHolderBuilder.cs

```diff
@@ -117,7 +117,7 @@ private void SetValidRotations(Vector3Int position, StaticSegmentData staticSegm
                     Rotation = rotation,
                     StaticSegmentData = staticSegmentData
                 })
-                .Where(segmentData => segmentData.StaticSegmentData.Reward > 0 
+                .Where(segmentData => segmentData.StaticSegmentData.SteamReward > 0 || segmentData.StaticSegmentData.BloodReward > 0
```

Comment:
Can you turn this predicate into a lambda variable in the static segment data class? so we can just call segmentData.IsReceiver. 

Would look like: 
bool IsReceiver => segmentData.StaticSegmentData.SteamReward > 0 || segmentData.StaticSegmentData.BloodReward > 0

That way the code becomes more readable.